{"name": "test_login[data1]", "status": "passed", "attachments": [{"name": "log", "source": "e5cb4ecb-1a59-4c8b-9909-212c82e5afc5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'case_id': 'login_case_002', 'case_title': '用户输入错误的用户名、\\n正确的密码，点击登录', 'test_data': {'username': 'alive1', 'password': 123456}, 'exp': {'by': 'xpath', 'locater': \"//*[@id='common-prompt']/div/p\", 'exp_value': '登录帐号不存在'}}"}], "start": 1663422093653, "stop": 1663422100875, "uuid": "9142085b-a564-411b-8141-35c4e8621d4c", "historyId": "6a42ab545080d6017bf21853c464e0cc", "testCaseId": "c6db3f0583de5245fe1a62b1d735a86c", "fullName": "run.TestLogin#test_login", "labels": [{"name": "suite", "value": "run"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-5FKMFO0H"}, {"name": "thread", "value": "25848-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "run"}]}